src/main/java/org/folio/edge/rtac/RtacHandler.java

```diff
@@ -102,7 +102,13 @@ protected void handle(RoutingContext ctx, boolean isBatch) {
               if (isBatch) {
                 returningContent = isXmlRequest ? instances.toXml() : instances.toJson();
               } else {
-                var holdings = instances.getHoldings().isEmpty() ? new Holdings() : instances.getHoldings().get(0);
+                var holdings = new Holdings();
+                if (instances.getHoldings().isEmpty() && !isXmlRequest) {
```

Comment:
Why is this check specific to only JSON requests?

Reply:
See above: Because the original issue asked for this to be added to the json response,
I had the code check for and not add the number to XML responses. It did not
seem to me to be a good idea to make changes to responses that were not
specifically asked for, given how many pieces of software use this module.

I've asked in the issue if this should be applied to the XML response as well.

Reply:
Original reporter confirmed the Id should be included in the XML response as well.  I've removed that part of the check, and updated the issue title and news file.